apps/website/src/pages/about/alveus.tsx

```diff
@@ -448,8 +448,8 @@ const AboutAlveusPage: NextPage = () => {
           </p>
         </div>
 
-        <div className="max-w-2xl basis-full p-4 pt-8 md:mx-0 md:basis-1/2 md:pt-4">
-          <Lightbox>
+        <div className="basis-full p-4 pt-8 xl:mx-0 xl:basis-1/2 xl:pt-4">
+          <Lightbox className="mx-auto max-w-2xl">
```

Comment:
Not sure about the mx-auto for larger viewports. Maybe only center it on the smaller breakpoints? In any case this the staff page should probably get the same treatment.

Reply:
What viewport looks odd to you with mx-auto? At larger viewports, there is very little margin that I can see and the mx-auto doesn't make much difference?

Reply:
For example at 1920 width. Tbh it does not really matter much. The site is great as it is. I mostly notice it in comparison with the staff page.

<table>
<tr>
 <td>
<img src="https://github.com/alveusgg/alveusgg/assets/684458/11058a76-04fb-42c3-9680-e15364a817e2">
 <td>
<img src="https://github.com/alveusgg/alveusgg/assets/684458/9681f827-79b0-4a6a-b12d-f440e4342108">
<tr>
 <td>Staff page, right aligned
 <td>About Alveus page, centered
</table>

Looking at it, maybe you could add the left padding to the other hero sections while you are at it?

Reply:
👍 I'll play around and try to get them all consistent